router/dealer.go

```diff
@@ -597,7 +602,7 @@ func (d *dealer) call(caller *Session, msg *wamp.Call) {
 		Arguments:    msg.Arguments,
 		ArgumentsKw:  msg.ArgumentsKw,
 	})
-	//log.Printf("dispatched CALL: %v [%v] to callee as INVOCATION %v",
+	//log.Printf("Dispatched CALL: %v [%v] to callee as INVOCATION %v",
```

Comment:
Yes, and a few other like it.  I left them in there in a commented out state, since I thought I may use them for debugging, but really, they should be removed. Will do.